### DISCUSSION TITLE
issues with run in background, ubuntu 20

### DISCUSSION THREAD

**Discussion by rsulym** [madMAx43v3r/chia-plotter#537] [Q&A]
Guys, does anybody making plots is background using 'screen' facility on ubuntu?
i have issues with background mode, jobs stuck sporadically on phases 1,2..

**Comment by aenciso** [comment]
Try using nohup e.g. `nohup ./build/chia_plot -n 1 -r 16 -t ...`

**Comment by kanasite** [comment]
been using screen on ubuntu, no issue so far

**Reply by rsulym** [reply]
what is your syntax? mine is
screen -d -m -S <job_name> bash -c 'plotter here | tee <log>'

**Comment by rsulym** [comment]
Guys, thanks for answers. I guess i have RAM overflow.
could you pls suggest proper approach to run 24 processes (i have 24 drives and CPU have 24 threads) for 128Gb of RAM?
